include/mantella_bits/numericalAnalysis.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+// C++ standard library
+#include <functional>
+
+// Armadillo
+#include <armadillo_bits/typedef_elem.hpp>
```

Comment:
This is something we don't make, but a common mistake if you use IWYU for the first time.

Armadillo needs to be *patched* for IWYU to recognise <armadillo> as the entry point:
`sudo find /usr/include/armadillo_bits -name *.hpp -exec sed -i -e '1i\/\/ IWYU pragma\: private\, include \<armadillo\>' {} ';'`
(assuming that you installed Armadillo under `user/include`)

This will also fix the
> error: ‘arma’ has not been declared
compiler error.